File: bootTest/Sample/src/main/java/sample/controller/HelloController.java
package sample.controller;

import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

//컨트롤러 이고 Rest 방식을 이용한다는 선언
@RestController
public class HelloController {
	//http://localhost:8080/
	@RequestMapping("/")
	public String hello() {
		return "안녕하세요!";
	}
	
}

File: Kotlin/Para/src/main/kotlin/org/example/para/controller/HelloController.kt
package org.example.para.controller

import org.example.para.common.ApiResponse
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class HelloController {

    @GetMapping("/api/vv/hello")
    fun hello() = ApiResponse.ok("world")


    @GetMapping("/hello")
    fun hello2(): String {
        return "hello"
    }
}